lib/core-server/src/utils/StoryIndexGenerator.test.ts

```diff
@@ -62,21 +63,23 @@ describe('StoryIndexGenerator', () => {
 
         expect(await generator.getIndex()).toMatchInlineSnapshot(`
           Object {
-            "stories": Object {
+            "entries": Object {
               "nested-button--story-one": Object {
                 "id": "nested-button--story-one",
                 "importPath": "./src/nested/Button.stories.ts",
                 "name": "Story One",
+                "storiesImports": Array [],
```

Comment:
I think this should be unset also for `type: 'story'` - or was there some reason we needed it?